src/widgets/auth/views/loginViewComponent.jsx

```diff
@@ -58,10 +58,15 @@ export const LoginForm = createForm({
 });
 
 export default class LoginView extends React.Component {
-    handleLogin = data => this.props.apiClient.loginWithPassword({
-        ...data,
-        ...pick(this.props, 'auth')
-    });
+    handleLogin = data => {
```

Comment:
Why did you do this change?

Reply:
I explained it on Slack. `persistent` has be passed inside `auth` field (`AuthOptions` type).
This code block also makes it overridable by editable options.

Reply:
If I do not merge both of the `auth` objects (one from the `component view form` and the other from `editable options`), then the one from `editable options` will superseed it entirely with the previous code.